packages/data-point/README.md

```diff
@@ -3542,6 +3543,53 @@ function resolve(acc:Accumulator, resolveReducer:function):Promise<Accumulator>
 
 Example at: [examples/custom-entity-type.js](examples/custom-entity-type.js)
 
+## <a name="tracing">Tracing DataPoint calls</a>
+
+To trace a DataPoint transformation you can set the value `trace:true` to the options object passed to the `datapoint.transform()` method. 
+
+This tool will write a JSON file to disk that exposes all of the reducers that are being executed within a transformation. It provides information such as time start and duration of each process as well as information regarding the reducer that was executed.
+
+### Example: 
+
+```js
+const DataPoint = require('data-point')
+const mocks = require('./async-example.mocks')
+
+// mock request calls
+mocks()
+
+const {
+  Model,
+  Request
+} = DataPoint.entities
+
+const PersonRequest = Request('PersonRequest', {
+  url: 'https://swapi.co/api/people/{value}/'
+})
+
+const PersonModel = Model('PersonModel', {
+  value: {
+    name: '$name',
+    birthYear: '$birth_year'
+  }
+})
+
+const options = {
+  trace: true 
+}
+
+const dataPoint = DataPoint.create()
+
+dataPoint
+  .transform([PersonRequest, PersonModel], 1, options)
+  .then((output) => {
+    // a file with the name data-point-trace-<timestamp>.json will
+    // be created.
+  })
```

Comment:
most of this example is not directly related to the `trace` option, so we should condense it. not sure how human readable the *-timestamp.json files are, but could we show an excerpt from one?